packages/@sanity/form-builder/src/inputs/PortableText/object/InlineObjectToolbarPopover.tsx

```diff
@@ -0,0 +1,112 @@
+import React, {useRef, useCallback, useEffect} from 'react'
+import {
+  Box,
+  Button,
+  Inline,
+  Popover,
+  PopoverProps,
+  Text,
+  useGlobalKeyDown,
+  useTheme,
+} from '@sanity/ui'
+import styled from 'styled-components'
+import {EditIcon, TrashIcon} from '@sanity/icons'
+
+const ToolbarPopover = styled(Popover)`
+  &[data-popper-reference-hidden='true'] {
+    display: none !important;
+  }
+`
+
+const POPOVER_FALLBACK_PLACEMENTS: PopoverProps['fallbackPlacements'] = ['top', 'bottom']
+
+interface InlineObjectToolbarPopoverProps {
+  onDelete: (event: React.MouseEvent<HTMLButtonElement>) => void
+  onEdit: (event: React.MouseEvent<HTMLButtonElement>) => void
+  open: boolean
+  referenceElement: HTMLElement
+  scrollElement: HTMLElement
+  setOpen: (open: boolean) => void
+  title: string
+}
+
+export function EditObjectToolTip(props: InlineObjectToolbarPopoverProps) {
```

Comment:
The component is exported as `EditObjectToolTip` but the file name is `InlineObjectToolbarPopoverProps` – should they be the same name?

Reply:
Most def! Renamed it. Thanks.